content/fringes/pulppluginday.md

```diff
@@ -2,20 +2,36 @@
 title: "Pulp Plugin Day"
 date: 2018-11-23
 eventdate: "6th February 2019"
-draft: true
+draft: false
 ---
 Pulp is a platform for managing repositories of software packages, but it can store any kind of
 binary data (e.g. cat photos). The core provides a REST API and manages files, but the real value
 for users is delivered by the plugins which allow core to manage a content type like a cat photo.
 
 Please join some of the Pulp developers on February 6th for Pulp Plugin Day. This will be an
-opportunity to add new features to any of the
+opportunity to learn how to use Pulp 3, add new features to any of the
 [existing plugins](https://pulpproject.org/pulp-3-plugins/) or start on a new plugin to support
 another content type.
 
-We are going to start at 10 am by going over the
-[Plugin Writer's Guide](https://docs.pulpproject.org/en/3.0/nightly/plugins/index.html#plugin-writer-s-guide).
-Then everyone is going to setup a development environment and start working on a Pulp plugin of
-their choice. 
-
 Please remember to [register](https://registration.cfgmgmtcamp.be/ghent/2019/) through the general Config Management Camp registration system.
+
+We are going to start at 10:00 with a brief introduction to Pulp. We will then help everyone setup
+a development environment on their laptop. At that point participants will decide what they would
+like to focus on. Some possible activities include:
+
+ * Fix an existing issue in [pulp.plan.io](https://pulp.plan.io/issues/)
```

Comment:
I think we should go into Redmine and ensure that there are good issue with the tags 'pulp3' + 'Easy Fix' and link to that here. Linking here now is fine, and someone can go fix the data closer to the event maybe? Also what about a link for 'groomed=True, state=NEW, tracker=[Story,Task]'. Anyway more specific Redmine links I think would be good.